packages/platform-ios/native_modules.rb

```diff
@@ -75,6 +76,7 @@ def use_native_modules!(root = "..", config = nil)
           phase["execution_position"] = phase["execution_position"].to_sym
         end
 
+        phase = Hash[phase.map { |k, v| [k.to_sym, v] }]
```

Comment:
cc @orta @alloy - I wasn't sure if this was the best way of doing it but I needed to symbolize the keys for it to be accepted for `script_phase` (otherwise errors with invalid args)

Reply:
I don't know if there's a _better_ way, but this is fine IMO